### PR TITLE
Disable TLS 1.3 by default

### PR DESCRIPTION
**Pull request by psiinon**
Also see https://github.com/zaproxy/zap-hud/issues/392

### CODE REVIEW

File: src/org/parosproxy/paros/network/SSLConnector.java [hunk 1]
```diff
@@ -34,6 +34,7 @@
 // ZAP: 2017/09/22 Rely on SNI if the domain is no known when creating the SSL/TLS tunnel.
 // ZAP: 2017/11/07 Pass the listeningIpAddress when creating the SSL/TLS tunnel. Check if hostname is an ipAddress when creating the SSL/TLS tunnel.
 // ZAP: 2018/06/08 Don't enable client cert if none set (Issue 4745).
+// ZAP: 1029/02/26 Disable TLS 1.3 by default as it currently fails with Java 11
```
Comment (thc202) [inline src/org/parosproxy/paros/network/SSLConnector.java]: The date needs a tweak.

File: src/org/parosproxy/paros/network/SSLConnector.java [hunk 2]
```diff
@@ -108,7 +109,9 @@
 		SECURITY_PROTOCOL_TLS_V1,
 		SECURITY_PROTOCOL_TLS_V1_1,
 		SECURITY_PROTOCOL_TLS_V1_2,
-		SECURITY_PROTOCOL_TLS_V1_3 };
+		// Disable TLS 1.3 by default as it currently fails with Java 11
```
Comment (thc202) [inline src/org/parosproxy/paros/network/SSLConnector.java]: Worth adding a task tag, to restore it once fixed.

File: src/org/parosproxy/paros/network/SSLConnector.java [hunk 3]
```diff
@@ -108,7 +109,9 @@
 		SECURITY_PROTOCOL_TLS_V1,
 		SECURITY_PROTOCOL_TLS_V1_1,
 		SECURITY_PROTOCOL_TLS_V1_2,
-		SECURITY_PROTOCOL_TLS_V1_3 };
+		// Disable TLS 1.3 by default as it currently fails with Java 11
+		// SECURITY_PROTOCOL_TLS_V1_3
```
Comment (thc202) [inline src/org/parosproxy/paros/network/SSLConnector.java]: This could be disabled just on the client side, the server side seems to be working fine.

**Comment by psiinon** [comment]
Hopefully fixed those comments :)

**Comment by thc202** [comment]
Thanks!

**Pull request merged**